yt/visualization/base_plot_types.py

```diff
@@ -334,10 +343,12 @@ def _init_image(self, data, cbnorm, cblinthresh, cmap, extent, aspect):
                         10
```

Comment:
Just so I understand, would it be possible to do this with `np.logspace`?  I do not think we should in this PR, for continuity sake, however.

Reply:
yes it should be possible to refactor this with np.logspace. I also agree that it's preferable to keep diffs minimal for a bugfix PR